source/Gesture Recognizer/Assets/Plugins/GestureRecognizer/Demo/Scripts/RecordGesture.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace GestureRecognizer.Demo
+{
+    public class RecordGesture : MonoBehaviour
```

Comment:
Make a parent class for `DetectGesture`, `ViewGesture` and `RecordGesture` to avoid repeating methods.

Reply:
Done